cmd/queue/main.go

```diff
@@ -265,8 +265,10 @@ func setupAdminHandlers(server *http.Server) {
 
 func main() {
 	flag.Parse()
-	logger, _ = logging.NewLogger(os.Getenv("SERVING_LOGGING_CONFIG"), os.Getenv("SERVING_LOGGING_LEVEL"))
+
+	logger, _ := logging.NewLogger(os.Getenv("SERVING_LOGGING_CONFIG"), os.Getenv("SERVING_LOGGING_LEVEL"))
```

Comment:
Why did this change? I'm curious if it works, since there are no new variables on the left side.

Reply:
Its defining a new variable `logger` and not assigning value to declared `logger` variable. I would expect the integration tests to fail.